src/OSPSuite.Core/Serialization/Chart/CurveChartXmlSerializer.cs

```diff
@@ -31,6 +32,18 @@ protected override void TypedDeserialize(TCurveChart curveChart, XElement output
 
    public class CurveChartXmlSerializer : CurveChartXmlSerializer<CurveChart>
    {
-      
+   }
+
+   public class SimulationPredictedVsObservedChartXmlSerializer : CurveChartXmlSerializer<SimulationPredictedVsObservedChart>
+   {
+      public override void PerformMapping()
+      {
+         base.PerformMapping();
+         Map(x => x.DeviationFoldValues);
+      }
+   }
+
+   public class SimulationResidualVsTimeChartSerializer : CurveChartXmlSerializer<SimulationResidualVsTimeChart>
```

Comment:
This serializer was missing. A bit puzzled here as to how it's being saved in PKSim